azuredevops/internal/acceptancetests/resource_wiki_test.go

```diff
@@ -0,0 +1,63 @@
+//go:build (all || wiki || resource_wiki) && !exclude_resource_wiki
+// +build all wiki resource_wiki
+// +build !exclude_resource_wiki
+
+package acceptancetests
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/microsoft/azure-devops-go-api/azuredevops/v7/wiki"
+	"github.com/microsoft/terraform-provider-azuredevops/azuredevops/internal/acceptancetests/testutils"
+	"github.com/microsoft/terraform-provider-azuredevops/azuredevops/internal/client"
+	"github.com/microsoft/terraform-provider-azuredevops/azuredevops/internal/utils/converter"
+)
+
+func TestAccWikiResource_CreateAndUpdate(t *testing.T) {
```

Comment:
Need more tests like `basic`, `update`, `complete`, `import` for this resource.
Ref: https://github.com/microsoft/terraform-provider-azuredevops/blob/main/azuredevops/internal/acceptancetests/resource_serviceendpoint_aws_test.go

Reply:
Hi @xuzhang3, is there any example for `import` test? refer to the [doc](https://developer.hashicorp.com/terraform/tutorials/providers-plugin-framework/providers-plugin-framework-resource-import), I don't quite understand how `acceptance test` and `plugin import implementation` works.
Or should I implement just `basic`, `update`, `complete` would be fine? Thanks.

Reply:
The import check is suit for resources that support import operation. Creating an existing resource will be rejected by the service and this test should detect his error.

Ref: https://github.com/microsoft/terraform-provider-azuredevops/blob/main/azuredevops/internal/acceptancetests/resource_serviceendpoint_bitbucket_test.go#L103

Reply:
I've tested the a scenario as below. The remote azure DevOps project has a wiki, then I use `terraform import azuredevops_wiki.test <wiki_id>`, the state has been update, then I need to sync the parameter of the `resource` in my test `main.tf`. So I'll try the `import` in acceptance test.

Reply:
```go
```

Reply:
btw, is the `import` test case design similar to [azurerm acceptance test](
https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/topics/reference-acceptance-testing.md#example---resource---requires-import)? I guess the goal might be the same as the `import` test case of this project.

Reply:
@chienfuchen32 yes, same purpose

Reply:
please confirm the change for me, thanks.